plugins/@grouparoo/demo/src/configFiles.ts

```diff
@@ -64,3 +69,28 @@ function updatePurchases(configDir) {
   const out = JSON.stringify(contents);
   fs.writeFileSync(appPath, out);
 }
+
+async function unlockAll() {
```

Comment:
I don't think I understand what this method is used for.  I'm guessing in the demo you want to use code-config to create all the objects but not have them "locked" so that they can be modified in the UI?

Is there a more general use-case for `locked = false` in code config?

Reply:
Yes, I wanted the demo to allow the user to change it.

It also makes it possible to do some of them at a time. It was necessary to run the `userId` rule and related source. Then run some event code. Then run the events config. If the lock was still there it was deleting the earlier ones.